src/sentry/static/sentry/app/styles/space.tsx

```diff
@@ -12,6 +12,8 @@ const SPACES = {
 
 export type ValidSize = keyof typeof SPACES;
 
-const space = (size: ValidSize): typeof SPACES[ValidSize] => SPACES[size];
+function space<Size extends ValidSize>(size: Size): typeof SPACES[Size] {
```

Comment:
I typically try and stick with the single character template variable names, but I'm fine either way.

Do you need the explicit return type?

Reply:
Looks like typescript does figure out the return type on its own.